### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by MaaooVaass**
## Travis-ci
functionality with travis

## Trello cards 
https://trello.com/c/tgcJDvVv/13-configurar-travisci

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+language: ruby
+rvm:
+  - 2.5.3
+
+addons:
```
Comment (lmasello) [inline .travis.yml]: No need for this one

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+env:
+  global:
+    - RUBY_GC_MALLOC_LIMIT=90000000
+    - RUBY_GC_HEAP_FREE_SLOTS=200000
+    - SECURITY_SECRET_TOKEN=traviscitraviscitraviscitraviscitravisci
+
+services:
+  - postgresql
+  #- redis-server
```
Comment (lmasello) [inline .travis.yml]: We usually remove the line instead of leaving it commented (the same for the others below)

**Review by lmasello — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  - gem update --system
+
+before_script:
+  - psql -c "CREATE USER \"wbooks-api\" WITH PASSWORD 'wbooks-api';" -U postgres
+  - psql -c "DROP DATABASE IF EXISTS \"WBook_test\";" -U postgres
+  - psql -c "CREATE DATABASE \"WBook_test\" ENCODING 'utf8';" -U postgres
+  - bundle exec rake db:migrate
+
+script:
+ # - bundle exec rspec spec -fd
```
Comment (fedeest) [inline .travis.yml]: Remove commented line

File: Gemfile [hunk 4]
```diff
@@ -1,5 +1,7 @@
+# frozen_string_literal: true
```
Comment (fedeest) [inline Gemfile]: Didn't you include the configuration for rubocop so that you don't have to do this evey time?

**Review by fedeest — CHANGES_REQUESTED** [review]
Small change

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+language: ruby
+
```
Comment (lmasello) [inline .travis.yml]: What about the removed rvm?

**Review by lmasello — APPROVED** [review]

**Review by fedeest — APPROVED** [review]

**Pull request merged**